frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -812,7 +819,8 @@ function NoNodeContextMenuOptions(props: NoNodeContextMenuProps) {
       mode="vertical"
     >
       {allActions}
-      {infoRows}
+      {/* In the TD viewport the info rows are not usable since the click position cannot be computed */}
+      {isTdViewport ? empty : infoRows}
```

Comment:
I think, it would be better if the caller of this function would pass an empty `infoRows` if the TD viewport is active. This would save some computations and allocations of JSX nodes and also the caller should have better knowledge over whether the info rows are irrelevant for the 3D case (e.g., in the future, there might be info rows which make sense for the 3D viewport).